src/App.tsx

```diff
@@ -1,14 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { TodoFilter } from './components/TodoFilter';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [selectStatus, setSelectStatus] = useState('all');
+  const [selectedTodo, setSelectedTodo] = useState<Todo>();
+  const [todoId, setTodoId] = useState(0);
+  const [load, setLoad] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig